snippets/code/web3py-tx/transaction.py

```diff
@@ -25,10 +25,10 @@
 )
 
 # Sign Tx with PK
-tx_create = web3.eth.account.signTransaction(
+tx_create = web3.eth.account.sign_transaction(
     {
-        "nonce": web3.eth.getTransactionCount(account_from["address"]),
-        "gasPrice": 0,
+        "nonce": web3.eth.get_transaction_count(account_from["address"]),
+        "gasPrice": web3.toWei("1", "gwei"),
```

Comment:
Would it be possible to get the gas price from chain? Some libraries have these methods